### DISCUSSION TITLE
Init a new project not working

### DISCUSSION THREAD

**Discussion by SunnyPirate** [quasarframework/quasar#10372] [CLI - PWA mode]
with option as below

sass with scss syntax
ESLint && TypeScript && Vuex && Axios
Quasar CLI requires a minor change to the root component:
/Users/qkb/Dev/test01/src/App.vue
Please add: id="q-app" (or write #q-app if using Pug)
to the outermost HTML element of the template.

Example:

**Comment by ontwikkelfabriek** [comment]
@SunnyPirate 

Please create a github **Issue** ( bug) if you think this is a...bug, in the **issue** section. This is the discussions forum.
